build/commands/lib/util.js

```diff
@@ -517,10 +517,24 @@ const util = {
     Log.progressFinish('touch original vector icon files overridden by brave/vector_icons')
   },
 
+  touchOverriddenPolicyScript: () => {
+    Log.progressStart('touch components/policy/tools/generated_policy_source.py if outdated')
```

Comment:
nit: I don't think we need to log such a small thing, the logging is more for long running process where we don't know what stage we're in. Perhaps we can catch any errors (if we expect any) and log there?

Reply:
Good idea, fixed and squashed in 91f6730749ab67c1b4a96c16caa3ccba5f7d3bb9